pages/about.js

```diff
@@ -2,16 +2,19 @@ import { Layout } from "../components/organisms/Layout";
 import Head from "next/head";
 import { serverSideTranslations } from "next-i18next/serverSideTranslations";
 import { useTranslation } from "next-i18next";
-import { useRouter } from "next/router";
 import { List } from "../components/molecules/List";
 import { CallToAction } from "../components/molecules/CallToAction";
 import { useEffect } from "react";
+import { useRouter } from "next/router";
 
 export default function About(props) {
   const { t } = useTranslation("common");
-  const { asPath } = useRouter();
+  const router = useRouter();
 
   useEffect(() => {
+    if (window.location.pathname === "/a-propos") {
+      router.push("/fr/a-propos");
+    }
```

Comment:
I really don't think this is a good idea, it would result in some people seeing the english page load, then disappear and  then the french page would show up - not a great user experience.

To fix this, in `ActionButton`, you can simply add `locale={props.lang}` in the `<Link>` and it will automatically do the language changing for you.